Python/CMakeLists.txt

```diff
@@ -13,6 +13,12 @@ else()
 # Use the latest C++ standard available for newer features
 set(PYBIND11_CPP_STANDARD /std:c++latest)
 endif()
+
+# ssize_t does not exist for MSVC but SSIZE_T is available in BaseTsd.
+ADD_DEFINITIONS(-Dssize_t=SSIZE_T)
+SET(CMAKE_REQUIRED_DEFINITIONS -Dssize_t=SSIZE_T)
+SET(CMAKE_EXTRA_INCLUDE_FILES BaseTsd.h)
+
```

Comment:
To be consistent with the rest of the file, can you change the cmake commands to lowercase? 

I still can't reproduce this problem on my end. I suspect that it has something to do with the cmake version on my machine. But the change looks good if it unblocks you we can take it.

Reply:
Done.